frontend/app/src/main/res/layout/activity_session.xml

```diff
@@ -6,4 +6,12 @@
     android:layout_height="match_parent"
     tools:context=".SessionActivity">
 
+    <TextView
+        android:id="@+id/session_key_text_view"
```

Comment:
we should be consistent w our namings - decide if we wanna do _text or _text_view or _textview - i think i did _text on mine